### ISSUE TITLE
Similar design for nema 23 and nema 34 wanted.

### ISSUE THREAD

**Issue by hemangjoshi37a** [creapunk/CLN17#3]
Can you please design anything similar for  nema 23 and nema 34 ?

**Comment by FutureEngineeer** [comment]
Already in progress)

**Comment by hemangjoshi37a** [comment]
@FutureEngineeer  Thank you sir. Please let me know if you  need any help. I can collaborate on this.

**Comment by Badwolf101** [comment]
I'm really interested in the project to use it on nema 14, I think the challenge is to design it for smaller motors, then just enlarge the size of the pcb

**Comment by ardlc-1212** [comment]
> I'm really interested in the project to use it on nema 14, I think the challenge is to design it for smaller motors, then just enlarge the size of the pcb

Not exactly... He would have to accomodate for bigger mosfets needed for the larger motors. Additionally there is hysteresis differences etc. that would require some additional testing

**Comment by thijstriemstra** [comment]
I'd also be very interested in a nema23 version for a clay extruder I'm building.
